src/components/look-controls.js

```diff
@@ -211,6 +232,17 @@ module.exports.Component = registerComponent('look-controls', {
   onMouseDown: function (event) {
     this.mouseDown = true;
     this.previousMouseEvent = event;
+
+    var sceneEl = this.el.sceneEl;
+    var canvasEl = sceneEl.canvas;
+
+    if (!this.pointerLocked) {
```

Comment:
if the user rejects the permission prompt, then for the rest of the page session, we probably shouldn't ever try to request Pointer Lock again.